lib/reports/subscriber_lists_report.rb

```diff
@@ -0,0 +1,70 @@
+class Reports::SubscriberListsReport
+  attr_reader :start_of_day, :end_of_day
+
+  CSV_HEADERS = %i[title
+                   slug
+                   matching_criteria
+                   created_at
+                   individual_subscribers
+                   daily_subscribers
+                   weekly_subscribers
+                   individual_updates_for_date
+                   unsubscribed].freeze
```

Comment:
This isn't necessary: it can be calculated from running the report over multiple days.

Reply:
True, you could but that would only show you the difference between the two dates after you totalled up the other subscriber types (bit of a chore).

However, mainly I'm seeing the value in a scenario where a list had lots of subscribing and unsubscribing then the amount of subscriptions could stay roughly the same even though theres lots of events. You would see this if you had an unsubscribe count but wouldn't without it.

Reply:
That's a good point. I was going to suggest this is less preferable, as you can't see what _kind_ of subscriptions were ended, but that applies equally based on what you said. 

Let's keep it then: hopefully we won't need to add more granular fields!